cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Connectors.java

```diff
@@ -83,6 +85,7 @@ private KafkaMirrorMaker2Connectors(Reconciliation reconciliation, KafkaMirrorMa
         this.mirrors = kafkaMirrorMaker2.getSpec().getMirrors();
         this.tracing = kafkaMirrorMaker2.getSpec().getTracing();
         this.rackAwarenessEnabled = kafkaMirrorMaker2.getSpec().getRack() != null;
+        //this.strimziMetricsReporterEnabled = kafkaMirrorMaker2.getSpec().getMetricsConfig() != null;
```

Comment:
You have to use `instanceof`, because it could be the other metrics type. Don't forget to enable all related code.